App.tsx

```diff
@@ -10,11 +10,12 @@ export default function App() {
   )
 }
 
+const backgroundColor = '#fff'
 const styles = StyleSheet.create({
   container: {
-    flex: 1,
-    backgroundColor: '#fff',
     alignItems: 'center',
+    backgroundColor,
+    flex: 1,
```

Comment:
question: was this sort from the eslint? Personally I don't like keys to be sorted automatically as it can also lead to bugs/unexpected behaviors. 

p.s. I didn't test locally.